tests/tests_vm_string.c

```diff
@@ -0,0 +1,194 @@
+#define MUNIT_ENABLE_ASSERT_ALIASES
```

Comment:
The same issues concerning the preprocessor and the inclusion of header files apply here.